oada-core/libs/oada-lib-arangodb/libs/exampledocs/authorizations.js

```diff
@@ -1,93 +1,102 @@
 module.exports = [
   {
-    '_id': 'authorizations/default:authorization-123',
-    'token': 'abc',
-    'scope': ['oada.yield:all', 'oada.rocks:all', 'oada.isoblue:all'],
-    'createTime': 1413831649937,
-    'expiresIn': 60,
-    'user': {'_id': 'users/default:users_frank_123'},
-    'clientId': 'jf93caauf3uzud7f308faesf3@provider.oada-dev.com'
+    _id: 'authorizations/default:authorization-123',
+    token: 'abc',
+    scope: ['oada.yield:all', 'oada.rocks:all', 'oada.isoblue:all'],
+    createTime: 1413831649937,
+    expiresIn: 60,
+    user: { _id: 'users/default:users_frank_123' },
+    clientId: 'jf93caauf3uzud7f308faesf3@provider.oada-dev.com'
   },
   {
-    '_id': 'authorizations/default:authorization-012',
-    'token': 'servio',
-    'scope': ['oada.fields:all', 'oada.operations:all'],
-    'createTime': 1413831649937,
-    'expiresIn': 60,
-    'user': {'_id': 'users/default:users_servio_012'},
-    'clientId': 'jf93caauf3uzud7f308faesf3@provider.oada-dev.com'
+    _id: 'authorizations/default:authorization-012',
+    token: 'servio',
+    scope: ['oada.fields:all', 'oada.operations:all'],
+    createTime: 1413831649937,
+    expiresIn: 60,
+    user: { _id: 'users/default:users_servio_012' },
+    clientId: 'jf93caauf3uzud7f308faesf3@provider.oada-dev.com'
   },
   {
-    '_id': 'authorizations/default:authorization-124',
-    'token': 'xyz',
-    'scope': ['oada.rocks:all'],
-    'createTime': 1413831649937,
-    'expiresIn': 60,
-    'user': {'_id': 'users/default:users_frank2_124'},
-    'clientId': 'jf93caauf3uzud7f308faesf3@provider.oada-dev.com'
+    _id: 'authorizations/default:authorization-124',
+    token: 'xyz',
+    scope: ['oada.rocks:all'],
+    createTime: 1413831649937,
+    expiresIn: 60,
+    user: { _id: 'users/default:users_frank2_124' },
+    clientId: 'jf93caauf3uzud7f308faesf3@provider.oada-dev.com'
   },
 
   {
-    '_id': 'authorizations/default:authorization-321',
-    'token': 'def',
-    'scope': ['oada.yield:all', 'trellisfw:all'],
-    'createTime': 1413831649937,
-    'expiresIn': 60,
-    'user': {'_id': 'users/default:users_sam_321'},
-    'clientId': 'jf93caauf3uzud7f308faesf3@provider.oada-dev.com'
+    _id: 'authorizations/default:authorization-321',
+    token: 'def',
+    scope: ['oada.yield:all', 'trellisfw:all'],
+    createTime: 1413831649937,
+    expiresIn: 60,
+    user: { _id: 'users/default:users_sam_321' },
+    clientId: 'jf93caauf3uzud7f308faesf3@provider.oada-dev.com'
   },
   {
-    '_id': 'authorizations/default:authorization-654',
-    'token': 'yyy',
-    'scope': ['oada.rocks:all'],
-    'createTime': 1413831649937,
-    'expiresIn': 60,
-    'user': {'_id': 'users/default:users_sam_321'},
-    'clientId': 'jf93caauf3uzud7f308faesf3@provider.oada-dev.com'
+    _id: 'authorizations/default:authorization-god',
+    token: 'god',
```

Comment:
Maybe not something to address here ... the `god` token is very useful in debugging, but it gets left behind in code and hides mediatype issues at development time.